packages/network/lib/index.ts

```diff
@@ -12,6 +13,7 @@ export {
   cors,
   httpUtils,
   uri,
+  networkPki,
```

Comment:
this is already in the `network` package, so this can be exported as just `pki`, not `networkPki`

Reply:
Hi @flotwig - we should be able to provide an update here early next week

Reply:
@flotwig Are you happy this is resolved in the latest commit? Can this conversation be resolved?